src/state_machine/signer/mod.rs

```diff
@@ -772,7 +772,16 @@ impl<SignerType: SignerTrait> Signer<SignerType> {
             self.decrypted_shares.insert(*src_id, decrypted_shares);
             self.decryption_keys
                 .insert(*src_id, (dkg_private_shares.signer_id, shared_key));
+            verified_shares.push((*src_id, encrypted_shares));
```

Comment:
This could result in empty encrypted shares hashmap per src_id. I assumed this was okay. but just confirming?